features/buyer/buyer_journey.feature

```diff
@@ -0,0 +1,26 @@
+@not-production @functional-test
+Feature: Buyer user journey through Digital Marketplace
+
+Scenario: Check the admin frontend /_status
+  Given I am logged in as a admin user
+  Then I am on the frontend /admin/_status page
+  Then I see 'ok' as the value of the 'status' JSON field
+  And Display the value of the 'version' JSON field as 'Release version'
+
+Scenario: Setup for tests
+  Given I have test suppliers
+  And The test suppliers have live services
+  And All services for the test suppliers are Public
+  Then I take a screenshot
+
+Scenario: There is pagination on the list of suppliers page if there are more than 100 results
+  Given I navigate to the list of 'Suppliers starting with S' page
+  When I click the 'Next page' link
+  Then I am taken to page '2' of results
+
+  When I click the 'Previous page' link
+  Then I am taken to page '1' of results
+
+Scenario: There is no pagination on the list of suppliers page if there is less than or equal to 100 results
+  Given I navigate to the list of 'Suppliers starting with Q' page
```

Comment:
This feels a little fragile. We have 51 suppliers on g-cloud-9 who's name starts with Q. I guess it's probably fine for quite some time but there may be a better way to do it.